app/models/miq_worker.rb

```diff
@@ -514,7 +514,7 @@ def minimal_class_name
 
   def database_application_name
     zone = MiqServer.my_server.zone
-    "MIQ #{Process.pid} #{minimal_class_name}[#{compressed_id}], s[#{miq_server.compressed_id}], #{zone.name}[#{zone.compressed_id}]".truncate(64)
+    "MIQ|#{Process.pid}|#{miq_server.compressed_id}|#{compressed_id}|#{zone.compressed_id}|#{minimal_class_name}|#{zone.name}".truncate(64)
```

Comment:
We might want to come back to this and figure out how to squash the provider workers names so it's not going to exceed 20 characters.